### PR TITLE
[stable/moodle] Simplify ingress configuration for cert-manager

### PR DESCRIPTION
**Pull request by juan131**
Signed-off-by: juan131 <juan@bitnami.com>

#### What this PR does / why we need it:

This PR allows the user to configure the proper annotations when using cert-manager to handle TLS certificates in an easy way. The user just needs to setup a boolean to true to do so.

### CODE REVIEW

File: stable/moodle/templates/ingress.yaml [hunk 1]
```diff
@@ -1,35 +1,40 @@
...
+  name: "{{- printf "%s-%s" .name $.Release.Name | trunc 63 | trimSuffix "-" -}}"
   labels:
-    app: {{ template "moodle.fullname" . }}
-    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
-    component: "{{ .Values.name }}"
-    heritage: {{ .Release.Service }}
-    release: {{ .Release.Name }}
-  name: {{ template "moodle.fullname" . }}
+    app: {{ template "moodle.fullname" $ }}
+    chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
```
Comment (tompizmor) [inline stable/moodle/templates/ingress.yaml]: `chart: {{ template "moodle.chart" $ }}`

File: stable/moodle/values.yaml [hunk 2]
```diff
@@ -139,32 +139,50 @@ mariadb:
...
-  hosts: []
-  #   - moodle.domain.com
+    ## Set this to true in order to enable TLS on the ingress record
+    ## A side effect of this will be that the backend moodle service will be connected at port 443
+    tls: false
 
-  ## Moodle server Ingress TLS configuration
-  ## Secrets must be manually created in the namespace
-  ## kubectl create secret tls moodle-server-tls --cert=tls.crt --key=tls.key
+    ## Set this to true in order to add the corresponding annontations for cert-manager
```
Comment (tompizmor) [inline stable/moodle/values.yaml]: typo

**Review by tompizmor — CHANGES_REQUESTED** [review]

File: stable/moodle/templates/ingress.yaml (hunk 1 above)
Comment (juan131) [reply stable/moodle/templates/ingress.yaml]: done

**Pull request merged**